lib/src/components/NxPageHeader/NxPageHeader.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import classnames from 'classnames';
+
+import { HeaderLinkProps, Props, propTypes } from './types';
+
+export { Props };
+
+const logoImg = require('../../assets/img/sonatype-logo-with-hexagon.png');
+
+function HeaderLink({ name, href, current }: HeaderLinkProps) {
+  const classes = classnames('nx-page-header__link', {
+    'nx-page-header__link--current': current
+  });
+
+  return (
+    <a className={classes} href={href}>
+      {/* See the comment in the CSS for why the name is here twice */}
+      <span className="nx-page-header__link-visible-text">{name}</span>
+      <span className="nx-page-header__link-sizing-text">{name}</span>
+    </a>
+  );
+}
+
+// visible for testing
+export { HeaderLinkProps, HeaderLink };
+
+export default function NxPageHeader({ links, homeLink, productInfo, children }: Props) {
+  const logoEl = <img src={logoImg} className="nx-product__wordmark" alt="⬡ Sonatype"/>,
```

Comment:
Clever, but I don't think the hexagon should be in the alt text

Reply:
> Clever, but I don't think the hexagon should be in the alt text

I don't feel too strongly about it, but after double checking the purpose of the alt text I felt like adding this was most faithful to it:

> Content associated with embedded content, to be used when the embedded content cannot itself be used (for example, when the embedded content is of an unsupported format).

Reply:
Ok